src/spacegame3/gamedata/systems/StarSystem.java

```diff
@@ -25,12 +25,15 @@ public StarSystem(Map<String, List<String>> data, CelestialBodyStructure cbs){
 
         List<String> systemData = data.get(ID_NAME);
 
-        for (String propraw : systemData){
+        for (String propraw : systemData)
+        {
             String[] prop = propraw.split("\\|");
-            switch (prop[PROP_NAME_POSITION]) {
+            switch (prop[PROP_NAME_POSITION])
+            {
```

Comment:
Don't change the coding style
also your change is mostly cosmetic, it adds nothing and may bring conflicts with other branches